.gitignore

```diff
@@ -30,6 +30,7 @@ npm-debug.log
 /android/titanium.bindings.json
 .idea/
 /android/.idea/
+/android/.idea/**
```

Comment:
Unrelated to this PR? It also should not be necessary, as the previous one includes sub-directories already. If you have modified files, you need to delete them once to be properly untracked